### PR TITLE
*: Update dependencies and fix unit test on Windows

### PR DESCRIPTION
**Pull request by kennytm**
### What problem does this PR solve? 

1. Update Go dependencies, move TiDB version to v3.1.0-beta
2. Update NPM dependencies for the web interface
3. Adjust the unit test so it passes on Windows.

### Check List 

Tests 

 - Unit test

Side effects

Related changes

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @suzaku @leoppro

File: lightning/common/util.go [hunk 1]
```diff
@@ -265,6 +265,12 @@ func GetJSON(client *http.Client, url string, v interface{}) error {
...
 // KillMySelf sends sigint to current process, used in integration test only
+//
+// Only works on Unix. Signaling on Windows is not supported.
 func KillMySelf() error {
-	return errors.Trace(syscall.Kill(syscall.Getpid(), syscall.SIGINT))
+	proc, err := os.FindProcess(os.Getpid())
+	if err == nil {
+		err = proc.Signal(syscall.SIGINT)
+	}
+	return errors.Trace(err)
```
Comment (suzaku) [inline lightning/common/util.go]: What's the difference between `Kill` and calling `Signal` on a process?

Comment (kennytm) [reply lightning/common/util.go]: [Basically the same](https://golang.org/src/os/exec_unix.go#L71), but this won't cause compilation failure outside Unix.

Comment (suzaku) [reply lightning/common/util.go]: TIL, thx.

**Comment by kennytm** [comment]
Ping @leoppro 

/run-all-tests

**Pull request merged**